apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowsTest.cs

```diff
@@ -282,5 +282,33 @@ public class MutateRowsTest
                         }
                     }));
         }
+
+        [Fact]
+        public async Task OmittedVersion()
+        {
+            var beforeInsertTimestamp = DateTime.UtcNow.AddMilliseconds(-1);
```

Comment:
Why the AddMilliseconds(-1) out of interest? The `<=` should cope if time is moving monotonically... and if we run the test across an NTP sync it could change by more than 1ms.

Or is this to avoid a rounding problem in converting between micros and DateTime?

Reply:
Yup, it's to avoid rounding issues. We round down to the millisecond because that's the granularity supported by Bigtable, so without backing up a millisecond, this was failing occasionally.

Reply:
Righto - makes perfect sense, thanks.